Subject: [PATCH 1/2] c_comment_scanner: fix function prototypes

scanner.l: Fix compile error as the following:

python3-hotdoc/0.17.4/sources/hotdoc-0.17.4/hotdoc/parsers/c_comment_scanner/scanner.l:126:1: error: conflicting types for 'parse_comment'; have 'int(PyObject *)' {aka 'int(struct _object *)'}
|   126 | parse_comment (PyObject *comments)

Upstream-Status: Backport [https://github.com/hotdoc/hotdoc/commit/adf8518431fafb78c9b47862a0a9a58824b6a421]

Signed-off-by: Liu Yiding <liuyd.fnst@fujitsu.com>
---
 hotdoc/parsers/c_comment_scanner/scanner.l | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hotdoc/parsers/c_comment_scanner/scanner.l b/hotdoc/parsers/c_comment_scanner/scanner.l
index 0408601..7bccd64 100644
--- a/hotdoc/parsers/c_comment_scanner/scanner.l
+++ b/hotdoc/parsers/c_comment_scanner/scanner.l
@@ -34,8 +34,8 @@
 extern int yylex (PyObject *comments);
 #define YY_DECL int yylex (PyObject *comments)
 static int yywrap (void);
-static int parse_comment ();
-static int parse_define ();
+static int parse_comment (PyObject *);
+static int parse_define (PyObject *);
 %}
 
 %option nounput
-- 
2.43.0

